### ISSUE TITLE
Uncaught Error: Data set 0 is invalid

### ISSUE THREAD

**Issue by dannykennedy** [Spyna/react-svg-radar-chart#22]
I'm getting the error `Uncaught Error: Data set 0 is invalid.` when I try to put less than 5 keys in the data field.
E.g. 
```
data: [
 {
  "data": {
   "battery": 0.7,
   "design": 1,
   "useful": 0.9
  },
  "meta": {
   "color": "#edc951"
  }
 },
 {
  "data": {
   "battery": 1,
   "design": 0.6,
   "useful": 0.8
  },
  "meta": {
   "color": "#cc333f"
  }
 },
 {
  "data": {
   "battery": 0.8,
   "design": 0.7,
   "useful": 0.6
  },
  "meta": {
   "color": "#00a0b0"
  }
 }
]
```

**Comment by dannykennedy** [comment]
As soon as there are 5 keys, e.g. 
```
 "data": {
   "battery": 0.7,
   "design": 1,
   "useful": 0.9,
   "speed": 0.67,
   "weight": 0.8
  },
```
It works fine...

**Comment by dannykennedy** [comment]
In fact, I'm just getting this error all over the place, and I can't work out what it means. What's invalid? I'm using Typescript and logging the different things, and nothing seems to be invalid. What can trigger this error??

**Comment by dannykennedy** [comment]
OK, ignore me, I'm stupid. I didn't realise that the captions (copy pasta) weren't matching the data. I'll leave this here for anyone else who's as silly as me. 

TL;DR, `Uncaught Error: Data set 0 is invalid` probably means `your data and captions don't match`.
